### ISSUE TITLE
maxLength values are wrong

### ISSUE THREAD

**Issue by clement911** [stripe/openapi#51]
It seems that most string fields have been give a `maxLength` of 5000 instead of a real value.

Here are just a couple of examples.

Currency code:
![image](https://user-images.githubusercontent.com/3426504/70327841-f885af00-1837-11ea-98b2-31c53d6fe794.png)

Object id:
![image](https://user-images.githubusercontent.com/3426504/70327872-0fc49c80-1838-11ea-9271-17c003df1d29.png)

[Docs state that](https://stripe.com/docs/upgrades#what-changes-does-stripe-consider-to-be-backwards-compatible): 

> You can safely assume object IDs we generate will never exceed 255 characters

For us, it's important to know the maxLength of fields when saving entities in a database to set a correct NVARCHAR or NCHAR length value in columns.

Any chance this will be fixed?

**Comment by remi-stripe** [comment]
@clement911 The spec does match what our API support today for legacy reasons. Some objects like `coupon` have a custom `id` field that you can set to more than 255 characters (and up to 5000). Changing this would be problematic since it could break existing integrations.

Our documentation specifically mentions that it's the values generated by Stripe for an `id` field will never exceed 255 characters.

**Comment by clement911** [comment]
These two were just examples but there seems to be many instances.

For example, the following fields also have a 5k max length:
email 
country
mcc (merchant category code)
phone
timezone
disabled_reason
zip code
etc... I'll stop there but there are tons of other.

Is the intent to really leave all of these at 5k?

**Comment by remi-stripe** [comment]
For now, the intent is to leave all of those to 5k yes and many of those fields do support those values. For a bit of historical context, many of our string values had literally no limit for a long time. When we started enforcing limits, we had to choose values that were reasonable while ensuring we don't break any existing integrations. For many fields, we had to settle on 5k because existing integrations were passing over 1k characters regularly.

We could have custom limits for every values, but it also adds overheard for developers to know that `zip_code`'s limit is 10 (and what about custom countries that have zip codes longer?) and that `email` can't be more than 100 characters, until someone writes in asking for a higher limit, etc.
